data_prep/introspector.py

```diff
@@ -39,64 +39,55 @@
 INTROSPECTOR_FUNC_SIG = f'{INTROSPECTOR_ENDPOINT}/function-signature'
 
 
+def _query_introspector(api: str, params: dict, key: str) -> Any:
+  """Queries FuzzIntrospector API and return data specified by |key|, returns None if unable to get the value."""
+  resp = requests.get(api, params, timeout=TIMEOUT)
+  if not resp.ok:
+    logging.error(f'Failed to get data from FI\n'
```

Comment:
nit: 
1. No need to use f-string here as it does not contain variables to format.
2. Use lazy formatting (instead of f-string) for logging (e.g., `logging.error('some text: %s', some_variable)`)

Same below.